File: calopy/src/calopy/maths/filter/DoNothingOnSeriesFilter.py
import logging

from calopy.maths.filter.Filter import Filter

DO_NOTHING = "Do nothing"


class DoNothingOnSeriesFilter(Filter):
    logger = logging.getLogger(__name__)
    type = DO_NOTHING

    def apply(self, series):
        print("Do nothing on series")
        return series

    def get_parameter_text(self):
        return ""

File: calojava/src/main/java/com/calopy/maths/filter/DoNothingOnSeriesFilter.java
package com.calopy.maths.filter;

import java.util.ArrayList;
import java.util.List;

public class DoNothingOnSeriesFilter implements CurveFittingFilter {

    public static final String TYPE = "Do nothing";

    @Override
    public List<Double> apply(List<Double> data) {
        return new ArrayList<>(data);
    }
}

